baselines/fedprox/pyproject.toml

```diff
@@ -141,3 +114,59 @@ exclude = [
 
 [tool.ruff.pydocstyle]
 convention = "numpy"
+
+[tool.hatch.build.targets.wheel]
+packages = ["."]
+
+[tool.flwr.app]
+publisher = "itsmaddox-j"
+
+[tool.flwr.app.components]
+serverapp = "fedprox.server_app:app"
+clientapp = "fedprox.client_app:app"
+
+[tool.flwr.app.config]
+[tool.flwr.app.config.algorithm]
+name = "FedProx"
+num_server_rounds = 100
+fraction_fit = 0.01
+fraction_evaluate= 0.0
+min_evaluate_clients= 0
+min_available_clients= 1000
+min_fit_clients= 1000
+local_epochs = 10
+stragglers_fraction = 0.9
+learning_rate = 0.03
+mu = 1.0
+num_clients = 1000 # This parameter is used by the Server only, and does not change the size of the federation.
+
+[tool.flwr.app.config.dataset]
+power_law = true
+num_unique_labels_per_partition = 2
+num_unique_labels = 10
+preassigned_num_samples_per_label = 5
+seed=42
+mu = 1.0
+sigma = 2.0
+val_ratio = 0.1
+batch_size = 10
+
+[tool.flwr.app.config.fit]
+drop_client = true # with FedProx, clients shouldn't be dropped even if they are stragglers
```

Comment:
```suggestion
drop_client = false # with FedProx, clients shouldn't be dropped even if they are stragglers
```

Reply:
Given that `name` is `FedProx` and `mu` is >0, we should then disable the dropping of clients?